package.json

```diff
@@ -9,12 +9,11 @@
   },
   "scripts": {
     "react-dev": "webpack -d --watch",
-    "bulbasaur": "node database/initializer.js",
+    "bulbasaur": "node database/index.js",
```

Comment:
This is a poorly named script. Variable naming is important in all your files, not just the js.